pkgs/tools/misc/partition-manager/default.nix

```
@@ -48,13 +48,16 @@ in mkDerivation rec {
     hash = "sha256-eChn3OkdLHC9pedDBBwszTeTj2l7ky2W79INqvjrkBo=";
   };
 
-  nativeBuildInputs = [ extra-cmake-modules kdoctools wrapGAppsHook wrapQtAppsHook makeWrapper ];
+  nativeBuildInputs = [ extra-cmake-modules kdoctools wrapGAppsHook ];
 
   propagatedBuildInputs = [ kconfig kcrash kinit kpmcore polkit-qt ];
 
+  dontWrapGApps = true;
   postFixup = ''
```

```suggestion
  preFixup = ''
```

Otherwise the wrapper args are never applied.

---

Ouch, `postFixup` is obviously too late and I only verified that the `XDG_*` variables are still in the wrapper.
the `GIO_*` ones indeed got lost, meaning `XDG_*` also come from Qt and not just GNOME.

Thanks, I'll amend this.